### PR TITLE
bpo-30361: Use better example for mixed-type operands

### PR DESCRIPTION
**Pull request by gfyoung**
Title is self-explanatory.  If this patch is accepted, I can also backport to the 3.6 branch.

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://cpython-devguide.readthedocs.io/pullrequest.html#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Comment by gfyoung** [comment]
> Unfortunately our records indicate you have not signed the CLA.

I just had it signed approximately an hour ago.  How long does it take to propagate?

File: Doc/tutorial/introduction.rst [hunk 1]
```diff
@@ -100,10 +100,8 @@ give you an error::
 There is full support for floating point; operators with mixed type operands
 convert the integer operand to floating point::
 
-   >>> 3 * 3.75 / 1.5
-   7.5
-   >>> 7.0 / 2
-   3.5
+   >>> 3 * 3.75 - 1
```
Comment (serhiy-storchaka) [inline Doc/tutorial/introduction.rst]: Wouldn't be better to use the example that takes the integer result? There is nothing unexpected that `3 * 3.75` results a `float`, it can't be an `int`. But `4 * 3.75` is an integer of type `float` even if it can be represented as `int`.

File: .gitignore [hunk 2]
```diff
@@ -1,8 +1,10 @@
 # Two-trick pony for OSX and other case insensitive file systems:
 # Ignore ./python binary on Unix but still look into ./Python/ directory.
+.idea/
```
Comment (serhiy-storchaka) [inline .gitignore]: Unrelated changes.

File: Doc/tutorial/introduction.rst (hunk 1 above)
Comment (gfyoung) [reply Doc/tutorial/introduction.rst]: Good point.  Fixed.

File: .gitignore (hunk 2 above)
Comment (gfyoung) [reply .gitignore]: Oops.  Will put in separate PR.

**Comment by gfyoung** [comment]
@serhiy-storchaka : Do you have any updates on this?

**Review by terryjreedy — APPROVED** [review]
